ribbon/ribbon/swap.py

```diff
@@ -165,12 +166,12 @@ def create_offer(self, offer: Offer, wallet: Wallet) -> str:
         offer.biddingToken = get_address(offer.biddingToken)
 
         nonce = self.w3.eth.get_transaction_count(wallet.public_key)
-        tx = self.contract.functions.createOffer(*list(asdict(offer).values())).build_transaction(
-            {
-                "nonce": nonce,
-                "gas": GAS_LIMIT,
-            }
-        )
+        if self.config.chain_id == Chains.BSC or self.config.chain_id == Chains.BSC_TESTNET:
```

Comment:
```suggestion
        if self.config.chain_id in [Chains.BSC, Chains.BSC_TESTNET]:
```

Reply:
We can do that, but considering the other check (FUJI & BSC & BSC_TESTNET) I'm not sure it solves so much

Here would be better:

```
- if self.config.chain_id in [Chains.BSC, Chains.BSC_TESTNET]:
+ if self.config.chain_id.is_binance:
```

But in the other case would be something like:

```
- if chain in [Chains.FUJI, Chains.BSC, Chains.BSC_TESTNET]:
+ if chain.is_binance or (chain.is_avalanche and chain.is_testnet):
```

Reply:
Agreed. So my two suggestions should be enough, thank you 🙏🏽